README.md

```diff
@@ -42,18 +50,33 @@ module.exports = function(environment) {
 `ember-cli-imgix` exposes a `img` element with expanded functionality. It simply renders an `img`, but has some extra parameters
 
 ```hbs
-{{imgix-image path='/users/1.png'}}
+{{imgix-image path='/users/1.png' sizes='100vw'}}
 ```
 
 Which will generate some HTML similar to this:
 
 ```html
-<img class="imgix-image" src="https://my-social-network.com/users/1.png?w=400&h=300&dpr=1" >
+<img
+  class="imgix-image"
+  src="https://my-social-network.com/users/1.png"
+  sizes="100vw"
+  srcset="https://my-social-network.com/users/1.png?w=100 100w, https://my-social-network.com/users/1.png?w=200 200w, ..."
+/>
 ```
 
 The src attribute will have imgix URL API parameters added to it to perform the resize.
 
-**Note:** This element works by calculating the width/height from its parent. If its parent has no width/height, then this component will do nothing.
+**Please note:** `100vw` is an appropriate `sizes` value for a full-bleed image. If your image is not full-bleed, you should use a different value for `sizes`. [Eric Portis' "Srcset and sizes"](https://ericportis.com/posts/2014/srcset-sizes/) article goes into depth on how to use the `sizes` attribute.
+
+Since imgix can generate as many derivative resolutions as needed, react-imgix calculates them programmatically, using the dimensions you specify. All of this information has been placed into the srcset and sizes attributes.
+
+**Width and height known:** If the width and height are known beforehand, it is recommended that they are set explicitly:
+
+```hbs
+{{imgix-image path='/users/1.png' sizes='100vw' width=100 height=100}}
```

Comment:
`sizes` shouldn't be required if both `width` and `height` are specified, right? If they're both provided, any reason we can't just automatically generate a DPR-style `srcset` and forget about `sizes`?